### PR TITLE
VZ-4709 Add doc for customizing AuthProxy and Istio components

### PR DESCRIPTION
**Pull request by mgianatagh**
Add doc for customizing AuthProxy and Istio components

### CODE REVIEW

**Review by rosemarymarano — COMMENTED** [review]
Few edits and suggestions

File: content/en/docs/setup/customizing/authproxy.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+The following table describes the fields in the Verrazzano custom resource pertaining to the AuthProxy configuration.
+
+| Name | Description |
+| --- | --- |
+| `spec.components.authProxy.kubernetes.replicas`  | The number of pods to replicate.  The default is 2 for the `prod` profile and 1 for all other profiles. |
+| `spec.components.authProxy.kubernetes.affinity`  | The pod affinity definition expressed as a standard Kubernetes [Affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity) definition.  The default configuration spreads the AuthProxy pods across the available nodes. <pre>spec:<br>  components:<br>    authProxy:<br>      kubernetes:<br>        affinity:<br>          podAntiAffinity:<br>            preferredDuringSchedulingIgnoredDuringExecution:<br>              - weight: 100<br>                podAffinityTerm:<br>                  labelSelector:<br>                    matchExpressions:<br>                      - key: app<br>                        operator: In<br>                        values:<br>                          - verrazzano-authproxy<br>                  topologyKey: kubernetes.io/hostname</pre> |
```
[the remark this answers is not in the document]
Comment (mgianatagh) [reply content/en/docs/setup/customizing/authproxy.md]: I changed it to lowercase

File: content/en/docs/setup/customizing/authproxy.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+The following is an example that customizes a Verrazzano `dev` profile as follows:
```
[the remark this answers is not in the document]
Comment (mgianatagh) [reply content/en/docs/setup/customizing/authproxy.md]: fixed

File: content/en/docs/setup/customizing/istio.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+title: "Customize Istio"
+description: "Customize Istio Gateways"
+linkTitle: Istio
+weight: 6
+draft: false
+---
+
+Verrazzano Istio gateways can be customized for High Availability by specifying replicas and affinity.
```
[the remark this answers is not in the document]
Comment (mgianatagh) [reply content/en/docs/setup/customizing/istio.md]: fixed

File: content/en/docs/setup/customizing/istio.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+The following table describes the fields in the Verrazzano custom resource pertaining to the Istio gateways configuration.
+
+| Name | Description |
+| --- | --- |
+| `spec.components.istio.egress.kubernetes.replicas`  | The number of pods to replicate.  The default is 2 for the `prod` profile and 1 for all other profiles. |
```
[the remark this answers is not in the document]
Comment (mgianatagh) [reply content/en/docs/setup/customizing/istio.md]: fixed

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md [hunk 5]
```diff
@@ -151,8 +165,30 @@ spec:
 ### Istio Component
 | Field | Type | Description | Required
 | --- | --- | --- | --- |
+| `enabled` | Boolean | If true, then Istio will be installed. | No |
+| `istioIngress` | [IstioIngress](#istio-ingress-configuration) | The Istio Ingress Gateway configuration. | No |
```
Comment (rosemarymarano) [inline content/en/docs/reference/API/Verrazzano/Verrazzano.md:169]: Istio Ingress Gateway -> Istio ingress gateway (globally)

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md [hunk 6]
```diff
@@ -151,8 +165,30 @@ spec:
...
+| `istioEgress` | [IstioEgress](#istio-egress-configuration) | The Istio Egress Gateway configuration. | No |
```
Comment (rosemarymarano) [inline content/en/docs/reference/API/Verrazzano/Verrazzano.md:170]: Istio Egress Gateway ->  Istio egress gateway (globally

**Review by kminder — APPROVED** [review]
Approved.

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md (hunk 5 above)
Comment (mgianatagh) [reply content/en/docs/reference/API/Verrazzano/Verrazzano.md]: fixed

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md (hunk 6 above)
Comment (mgianatagh) [reply content/en/docs/reference/API/Verrazzano/Verrazzano.md]: fixed

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**